androidHyperskillApp/src/main/java/org/hyperskill/app/android/step/view/fragment/StepFragment.kt

```diff
@@ -35,7 +36,7 @@ class StepFragment : Fragment(R.layout.fragment_step), ReduxView<StepFeature.Sta
     private val stepViewModel: StepViewModel by reduxViewModel(this) { viewModelFactory }
     private val viewStateDelegate: ViewStateDelegate<StepFeature.State> = ViewStateDelegate()
 
-    private var stepId: Long by argument()
+    private var stepRoute: StepRoute by argument()
```

Comment:
Чтобы сохранить комплексный объект в аргументы фрагмента, нужно использовать функцию, которая принимает serializer (Объект должен быть помечен как @Serializable)
```suggestion
    private var stepRoute: StepRoute by argument(serializer = StepRoute.serializer())
```

Reply:
Забыл добавить, что эта функция в develop ветке лежит, нужно подтянуть изменения оттуда